src/gt4py/next/iterator/type_inference.py

```diff
@@ -387,7 +394,14 @@ def _infer_shift_location_types(shift_args, offset_provider, constraints):
 
 @dataclasses.dataclass
 class _TypeInferrer(eve.traits.VisitorWithSymbolTableTrait, eve.NodeTranslator):
-    """Visit the full iterator IR tree, convert nodes to respective types and generate constraints."""
+    """
+    Visit the full iterator IR tree, convert nodes to respective types and generate constraints.
+
+    Attributes:
+        collected_types Mapping from the (Python) id of a node to its type.
+        constraints     Set of constraints, where a constraint is a pair of types that need to
```

Comment:
Please use the correct Google style format for the docstrings (as required by sphix-napoleon: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google)
```suggestion
        collected_types: Mapping from the (Python) id of a node to its type.
        constraints: Set of constraints, where a constraint is a pair of types that need to
```

Reply:
Fixed. This should be adopted in the https://github.com/GridTools/gt4py/blob/main/CODING_GUIDELINES.rst at some point.

Reply:
It was already there in functional (https://github.com/GridTools/gt4py/blob/functional/CODING_GUIDELINES.md#docstrings), but it got lost during the merge. I will update soon the documents in master with the versions from the other branch.